src/main/java/org/folio/circulation/domain/OpeningPeriod.java

```diff
@@ -0,0 +1,50 @@
+package org.folio.circulation.domain;
+
+import io.vertx.core.json.JsonObject;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.LocalDate;
+import org.joda.time.LocalTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+public class OpeningPeriod {
+  private static final String OPENING_DAY_KEY = "openingDay";
+  private static final String DATE_KEY = "date";
+  private static final String DATE_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
+  private static final DateTimeFormatter DATE_TIME_FORMATTER =
+    DateTimeFormat.forPattern(DATE_TIME_FORMAT).withZoneUTC();
+
+  private OpeningDay openingDay;
+  private LocalDate date;
```

Comment:
What does this field represent, what is it the date of or for?

Reply:
It represents the date an opening day is referring to. Here is an example of response from `/calendar/periods`:
"openingPeriods": [
    {
      "openingDay": {
        "openingHour": [
          {
            "startTime": "07:00",
            "endTime": "13:00"
          }
        ],
        "allDay": false,
        "open": true,
        "exceptional": false
      },
      "date": "2020-01-03T00:00:00.000+0000"
    }
  ],
  "totalRecords": 1
}

Reply:
I don't think the production domain code should include code used for constructing mock responses (which I would prefer worked differently in general)

If there is a need to construct fake records specifically in the test infrastructure, I think that should be done with test specific code.